core/src/main/java/org/apache/carbondata/core/datamap/Segment.java

```diff
@@ -36,9 +39,23 @@
 
   private String segmentFileName;
 
+  private ReadCommittedScope readCommittedScope;
```

Comment:
please provide comment for this variable

Reply:
Done